src/cephadm/cephadm

```diff
@@ -365,9 +366,9 @@ class CephIscsi(object):
     def get_container_mounts(data_dir, log_dir):
         # type: (str, str) -> Dict[str, str]
         mounts = dict()
-        mounts[os.path.join(data_dir, 'config')] = '/etc/ceph/ceph.conf:z'
-        mounts[os.path.join(data_dir, 'keyring')] = '/etc/ceph/keyring:z'
-        mounts[os.path.join(data_dir, 'iscsi-gateway.cfg')] = '/etc/ceph/iscsi-gateway.cfg:z'
+        mounts[os.path.join(data_dir, 'config')] = ETC_CEPH_DIR + '/ceph.conf:z'
```

Comment:
totally not sure about this one. this is `/etc/ceph/ceph.conf`, but it has nothing to do with SHELL_DEFAULT_CONF, as is lives inside the container, which is a different universe. 

Also ETC_CEPH_DIR is it supposed to be within the container universe or outside or both?